src/app/pages/Swap/hooks/use-tkey-ad.ts

```diff
@@ -1,17 +0,0 @@
-import { useMemo } from 'react';
-
-import { useLocation } from 'lib/woozie';
-
-import { EXTERNAL_AD_QUERY } from '../constants';
-
-export const useTKeyAd = () => {
```

Comment:
Ask Dima if we will support redirect to swap page from tkey ads. If no, we can remove some more code